src/App.jsx

```diff
@@ -14,33 +16,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState(false);
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  let visibleGoods = goodsFromServer.toSorted((good1, good2) => {
+    switch (sortField) {
+      case 'Sort alphabetically':
+        return good1.localeCompare(good2);
+      case 'Sort by length':
+        return good1.length - good2.length;
+      case 'Reset':
```

Comment:
Create a variable for these words `'Sort alphabetically', 'Sort by length', 'Reset'` and use it everywhere